### PR TITLE
@0x48piraj : Added self-composed beat for background music

### PR DESCRIPTION
**Pull request by 0x48piraj**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- Adds background digital music composed by **Piyush Raj**

### CODE REVIEW

**Comment by 0x48piraj** [comment]
Ping @niccokunzmann :heart:

File: js/sound.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 /*jshint esversion: 6 */
 const backgroundAudio = [
+    {
+        filename: "audio/0x48piraj/composed/0x48piraj_Rusted_Heart.mp3",
```
Comment (abishekvashok) [inline js/sound.js]: I cannot find this in the diff

Comment (0x48piraj) [reply js/sound.js]: I have added this in a previous Pull Request :+1:

**Comment by niccokunzmann** [comment]
Thanks for the reviews!

**Comment by niccokunzmann** [comment]
Thanks for the PR!

**Pull request merged**